packages/edit-post/src/components/preview-options/style.scss

```diff
@@ -53,11 +44,16 @@
 }
 
 .editor-post-preview__button-external {
-	padding-left: $button-size-small + $grid-unit-10 + $grid-unit-10;
+	padding-left: 8px;
 	margin-right: auto;
 	width: 100%;
 	display: flex;
-	justify-content: space-between;
+	justify-content: flex-start;
+
+	svg {
+		margin-right: 8px;
```

Comment:
<img width="227" alt="Screenshot 2020-03-13 at 14 10 26" src="https://user-images.githubusercontent.com/1204802/76625216-0b92d400-6537-11ea-90bb-fa93bb4d9082.png">

This caused a few issues, so let's discuss this separately. In the mean time I removed it.